robottelo/constants/repos.py

```diff
@@ -52,3 +52,6 @@
     'https://fixtures.pulpproject.org/rpm-string-version-updateinfo/'
 )
 EPEL_REPO = f'{REPOS_URL}/epel_repo'
+
+ANSIBLE_GALAXY = 'https://galaxy.ansible.com/'
```

Comment:
I think it would be better to use these as Dynaconf validator defaults instead of constants. That way its actually configurable, but doesn't have to be set in the yaml.

Reply:
That's an interesting case. How often likely is it that we would be able to deviate from this value?

Reply:
Well, we will have our own galaxy mirror via pulp before too long. We could have more control over what content is available to test against.

Thoughts/Notes for future updates, not something that needs to be addressed now.